recipes-core/acrn/acrn-hypervisor.bb

```diff
@@ -16,14 +16,11 @@ DEPENDS += "python3-kconfiglib-native"
 DEPENDS += "${@'gnu-efi' if d.getVar('ACRN_FIRMWARE') == 'uefi' else ''}"
 
 do_configure() {
-	mkdir --parents ${B}/hypervisor
-	cat <<-EOF >> ${B}/hypervisor/.config
-CONFIG_BOARD="${ACRN_BOARD}"
+		cat <<-EOF >> ${S}/hypervisor/arch/x86/configs/${ACRN_BOARD}.config
 CONFIG_$(echo ${ACRN_SCENARIO} | tr '[:lower:]' '[:upper:]')=y
```

Comment:
I can see we are not running with oldconfig. Is CONFIG_xxx=y for scenario still required to append in .config.

Reply:
yes, else it would always be default scenario